michaelplazek/on-track
# Fixed clock and disabled features for Moving Block Mode

## Description
This includes the following features:
- buttons/dropdowns/tables are now disabled when Moving Block Mode is selected
- "Time" column has been removed from the Add Train schedule table
- the clock now works perfectly

## Changes
N/A

## Testing
To test Moving Block Mode:
- select the radio button in the top right-hand corner
- confirm that everything is disabled

To test clock: do whatever you want - it's unbreakable!

## Additional Information
None.
